src/backend/distributed/planner/multi_explain.c

```diff
@@ -432,8 +447,20 @@ ExplainSubPlans(DistributedPlan *distributedPlan, ExplainState *es)
 
 		ExplainOpenGroup("PlannedStmt", "PlannedStmt", false, es);
 
+#if PG_VERSION_NUM >= PG_VERSION_17
+		if (es->memory)
```

Comment:
Github issue should be created to track supporting MEMORY option when extracting EXPLAIN output from workers. 
https://github.com/citusdata/citus/blob/f6959715dc49404510f2f3340b9719b33ddd152b/src/backend/distributed/planner/multi_explain.c#L1098

Nothing should fail now.  But we will not be showing memory usage from workers if we do not fix this.

Reply:
Also adding this here https://github.com/citusdata/citus/issues/7653